detector/ignores_test.go

```diff
@@ -73,6 +73,31 @@ func TestDirectoryPatterns(t *testing.T) {
 	assertDenies("foo/", "filename", "foo/bar/baz.txt", t)
 }
 
+func TestIgnoreAdditionsByScope(t *testing.T) {
+	file1 := testAddition("yarn.lock")
+	file2 := testAddition("similaryarn.lock")
+	file3 := testAddition("java.lock")
+	file4 := testAddition("Gopkg.lock")
+	file5 := testAddition("vendors/abc")
+	additions := []git_repo.Addition{file1, file2, file3, file4, file5}
+
+	scopesToIgnore := []string{"node", "go"}
+	talismanRCIgnoreConfig := CreateTalismanRCIgnoreWithScopeIgnore(scopesToIgnore)
+
+	nodeIgnores := []string{"node.lock", "yarn.lock"}
+	javaIgnores := []string{"java.lock"}
```

Comment:
Is java.lock a thing ? Or is this dummy test data ?

Reply:
It's dummy test data